### ISSUE TITLE
Editor control has no description.

### ISSUE THREAD

**Issue by briancwelch** [themeum/kirki#139]
```
    $controls[] = array(
        'type'        => 'editor',
        'setting'     => 'projects_html_after',
        'label'       => __( 'Add/Edit Projects - Custom HTML (After)', 'maera-diy' ),
        'description' => __( 'You can use this field to enter any custom HTML that you would like to be displayed after the Add/Edit projects fields in the popup window.', 'maera-diy' ),
        'default'     => '<h3>My Projects</h3>',
        'priority'    => 3,
        'section'     => 'maera_diy_general',
    );
```

Visual Reference: 
![capture](https://cloud.githubusercontent.com/assets/3002855/6886724/a83c2088-d615-11e4-8793-618a05fa8303.PNG)

**Comment by stev0n** [comment]
Hi
Has support for having html as the default text been removed ?

**Comment by aristath** [comment]
@stev0n HTML is stripped for security reasons, yes.

**Comment by stev0n** [comment]
Hi

Thanks for your eply.

Is it not possible to just drop out the javascript tags and leave the others ?

regards
    steve

**Comment by aristath** [comment]
Fixed.
If you pull a fresh copy it should now be working fine.
Thanks for reporting this!
However, please try to create a new ticket for issues like that so that we may better keep track of them. I almost missed the comments above since this ticket was already closed. :+1:

**Comment by stev0n** [comment]
Hi Aristeides

Thanks, this works. I didn’t realise it was a bug and not a choice for security reasons.

But I had to make a little change.

if('' != $this->help )  —>   if ( ! empty($this->help )) 
if help not set in the customiser, you get an error.

also,  I only downloaded this one page (class-kirki-controls-editor-control.php ) , so i had to change
extends Kirki_Customize_Control  —>  extends WP_Customize_Control 

I think this project is great.
Recently, i had to write a custom control to handle a group of number fileds, which i wanted stored in the database as key -> pairs.
I did manage this with a little custom js and a modification to the class file so that it could  handle either the default key -> pair array or the serialized / json  (string) which my custom class recieves from the db.
The ability to handle key -> pair data in and out of the db would be a useful enhancement for your other controls. i could provide my scirpts if you are interested.
 is there a place where you would like me to post a request for this upgrade ?

keep up the good work

regards
    steve
